app/views/businessmatching/updateservice/remove/RemoveActivitiesView.scala.html

```diff
@@ -50,7 +50,8 @@
 
     @formHelper(action = controllers.businessmatching.updateservice.remove.routes.RemoveBusinessTypesController.post(edit)) {
 
-        @errorSummary(form, Some(Map("value" -> "value_0")))
+        @errorSummary(form, Some(Map("value" -> s"value_${(BusinessActivities.all.toSeq diff activityValues).map(activity =>
```

Comment:
I'm not sure this is working as intended. I just checked this page for user XBML00000567890 and the error summary takes my focus to the middle of the list, the HVD option aka `#value_5`

Reply:
Will have a look now